LongWarOfTheChosen/Src/LW_XModBase/Classes/XMBEffect_ConditionalBonus.uc

```diff
@@ -230,13 +230,18 @@ static function name ValidateWeapon(ExtShotModifierInfo ExtModInfo, XComGameStat
 	return 'AA_Success';
 }
 
-// From X2Effect_Persistent. Returns a damage modifier for an attack by the unit with the effect.
-function int GetAttackingDamageModifier(XComGameState_Effect EffectState, XComGameState_Unit Attacker, Damageable TargetDamageable, XComGameState_Ability AbilityState, const out EffectAppliedData AppliedData, const int CurrentDamage, optional XComGameState NewGameState)
+function float GetPostDefaultAttackingDamageModifier_CH(
```

Comment:
Will these changes get overridden if we upgrade XModBase again?

Reply:
Yes, but I don't think we'll need to upgrade it.